string-variants.cabal

```diff
@@ -5,7 +5,7 @@ cabal-version: 1.12
 -- see: https://github.com/sol/hpack
 
 name:           string-variants
-version:        0.3.1.1
+version:        0.3.1.3
```

Comment:
oh i think you missed a package.yaml...

Reply:
https://github.com/MercuryTechnologies/string-variants/pull/27